Serverless/Azure Functions/About Azure Functions.md

```diff
@@ -1,36 +1,57 @@
+The default function app consists of:
 
+* `Function.cs` class - Contains a function method with defined triggers and input/output bindings. The `[FunctionName]` attribute marks the method as a function entry point. The name must be unique within a project.
+* `local.settings.json` - Contains all app settings, connection strings, and configurations. Settings in the `local.settings.json` file are used only when you're running the project locally so it is not deployed. Note that when you create a new Function project, this file is added to the `.gitignore` file, meaning that it will not end up in the repository if we don't set that up ourselves. By default, it contains the ``AzureWebJobsStorage`` key with a value set to ``UseDevelopmentStorage=true``, which is the connection string for the local Azure Storage Emulator.
+* `host.json` - Contains the global config options for all functions within a Function app. Here you can configure logging and retry logic. See options in detail [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json#sample-hostjson-file).
+* `Program.cs`  (applicable for out-of-process) - Contains host builder logic, DI configuration and middleware configuration.
 
 ### Function Properties
 
-* Short-lived - Serverless functions should be short-lived and stateless. The default lifetime is 5 minutes with a maximum of 10 minutes. If the function execution time exceeds the configured lifetime it will be un-gracefully killed.
-* Retry logic - Retry policies can be defined for all functions in an app by setting properties in host.json or for individual functions using attributes.
-Retry options are: ``fixedDelay`` or ``ExponentialBackoff``. Learn more [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-error-pages?tabs=csharp#retry-policies-preview).
+#### Lifetime
+
+Serverless functions should be short-lived and stateless. The maximum lifetime of a function can be configured using the `functionTimeout` property. The default lifetime is 5 and 30 minutes with a maximum of 10 or unlimited for Consumption and other plans respectively. If the function execution time exceeds the configured lifetime it will be un-gracefully killed.
+
+``` json
+{
+	"functionTimeout": "00:05:00"
+}
+```
+
+#### Retry logic
+
+[Retry policies](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-error-pages?tabs=csharp#retry-policies-preview) can be defined for all functions in an app by setting properties in host.json or for individual functions using attributes. Retry options are:
+
+- Fixed delay - a specified amount of time is allowed to elapse between each try.
+- Exponential backoff - the amount of time between each try increases exponentially
```

Comment:
First option (`fixed delay`) has a dot at the end, the second one (`exponential backoff`) doesn't, we should probably follow the same convention 😄